docs/docs-content/clusters/pcg/manage-pcg/registry-override.md

```diff
@@ -0,0 +1,195 @@
+---
+sidebar_label: "Override Registry Configuration"
+title: "Override Registry Configuration"
+description: "Learn how to override the image registry configuration for a Private Cloud Gateway (PCG) in Palette."
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["pcg"]
+---
+
+You can override the image registry configuration for a Private Cloud Gateway (PCG) to use a custom image registry for
+the PCG cluster. This feature is useful when you want to use a custom image registry to store and manage the images used
+by the PCG cluster. The image registry configuration is applied to the PCG cluster and is used to pull the required
+images for the PCG cluster.
+
+## Prerequisites
+
+Before you override the image registry configuration for a PCG, ensure you have the following:
+
+- An active PCG cluster. Refer to [Deploy a PCG](../deploy-pcg/deploy-pcg.md) to learn how to deploy a PCG.
+
+- Access to the kubeconfig file for the PCG cluster. You need the kubeconfig file to access the PCG cluster and apply
+  the image registry configuration.
+
+  :::tip
+
+  You can download the kubeconfig file from the PCG cluster details page in Palette. Navigate to the PCG cluster details
+  page. Click on the **Admin Kubeconfig** link to download the kubeconfig file. If you need help with configuring
+  kubectl to access the PCG cluster, refer to the [Access Cluster with CLI](../../cluster-management/palette-webctl.md)
+  guide.
```

Comment:
```suggestion
  You can download the kubeconfig file from the PCG cluster details page in Palette:
  - Navigate to the PCG cluster details
  page and click on the **Admin Kubeconfig** link to download the kubeconfig file.
  
  If you need help with configuring
  kubectl to access the PCG cluster, refer to the [Access Cluster with CLI](../../cluster-management/palette-webctl.md)
  guide.
```

Reply:
Just a thought, but we could structure this as a one-step procedure, in which case we just go with a bullet point. I'm using the Microsoft Style Guide as reference here ([Single-step procedures](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions#single-step-procedures)), so it's absolutely optional